web3/pm.py

```diff
@@ -479,7 +485,7 @@ def get_all_package_releases(self, package_name: str) -> Iterable[Tuple[str, str
             _, version, manifest_uri = self.registry._get_release_data(release_id)
             yield (version, manifest_uri)
 
-    def get_release_id_data(self, release_id: bytes) -> Tuple[str, str, str]:
+    def get_release_id_data(self, release_id: bytes) -> Tuple[str, ...]:
```

Comment:
This return type doesn't match the docstring (or my understanding of what this API returns).  Why isn't it `Tuple[str, str, str]` anymore?

Reply:
This is related to [`_get_release_data() -> Iterable[str]`](https://github.com/ethereum/web3.py/pull/1507/files#diff-b76a8371a5c2d30f739ddcd21e7a6b95R168) b/c of using the `@to_tuple` decorator.

Reply:
If this is a fixed size Tuple then it would be better to not use `@to_tuple` but rather return the tuple manually (`return (x, y, z)`). The `to_tuple` decorator will always just convert a `Iterable[Something]` into a `Tuple[Something, ...]` but that doesn't convey well the intention of the API if the value is always of the same size. 

In fact, you *could* even consider creating a `NamedTuple` such as `ReleaseData` to get a stronger API. That would also eliminate the need for placeholder variables when you are only interested in one of the three values.

E.g. instead of.

```python
_, __, what_i_need = _get_release_data()
print(what_i_need)
```

You could just write

```python

release_data = _get_release_data()
print(release_data.what_i_need)
```

Reply:
Ahh, that's a good point. Made those updates, danke @cburgdorf !!

Reply:
Gerne :)